docs/FAQs.md

```diff
@@ -93,7 +93,7 @@ Here we'll outline the steps required to migrate a CTA app to a GitHub Action:
    ```diff
    +pnpm run build
    +git add dist
-   npx lint-staged
+   pnpx lint-staged
```

Comment:
This *could* be reverted, but I kept this change because `pnpx` is just better. It's faster and what the project will be using after this commit.

Reply:
> faster

Is it, though? Trying out an arbitrary text addition of `\nExample.\n` to the bottom of `README.md`:

```plaintext
joshgoldberg ~/repos/create-typescript-app $ hyperfine "npx lint-staged" --warmup 3
Benchmark 1: npx lint-staged
  Time (mean ± σ):     843.7 ms ±   9.0 ms    [User: 833.9 ms, System: 206.5 ms]
  Range (min … max):   832.7 ms … 864.0 ms    10 runs
 
joshgoldberg ~/repos/create-typescript-app $ hyperfine "pnpx lint-staged" --warmup 3
Benchmark 1: pnpx lint-staged
  Time (mean ± σ):     974.1 ms ±  22.9 ms    [User: 854.3 ms, System: 185.1 ms]
  Range (min … max):   943.5 ms … 1007.2 ms    10 runs
```

System info:
- OS: macOS Ventura 13.2
- Hardware: Mac Studio, Apple M1 Max, 32 GB
- Node: 22.12.0
- npm: 10.9.0
- pnpm: 10.5.2

Reply:
It is not! My mistake!
```sh
> hyperfine "npx lint-staged" "pnpx lint-staged" --warmup 3
Benchmark 1: npx lint-staged
  Time (mean ± σ):     691.0 ms ±   6.2 ms    [User: 815.1 ms, System: 161.2 ms]
  Range (min … max):   685.7 ms … 706.3 ms    10 runs

Benchmark 2: pnpx lint-staged
  Time (mean ± σ):      1.098 s ±  0.047 s    [User: 0.879 s, System: 0.152 s]
  Range (min … max):    1.054 s …  1.184 s    10 runs

Summary
  npx lint-staged ran
    1.59 ± 0.07 times faster than pnpx lint-staged
```